api/swagger.yaml

```diff
@@ -2764,6 +2764,16 @@ definitions:
             type: "object"
             additionalProperties:
               type: "string"
+          MemorySwap:
+            description: "Total memory limit (memory + swap). Set as `-1` to enable unlimited swap."
```

Comment:
`s/Set as/Set to/`
(I know there's another similar sentence in there, but "Set to" prevails)